examples/qvgviewer/CMakeLists.txt

```diff
@@ -0,0 +1,45 @@
+list(APPEND TARGET_HEADERS
+    MainWindow.h)
+
+list(APPEND TARGET_SOURCES
+    MainWindow.cpp
+    main.cpp)
+
+set(TARGET_NAME qvgviewer)
+
+#CONFIG += autoqvg
+
+option(autoqvg "enable automatic qvg commpilation" OFF)
```

Comment:
@uwerat Should the user be able to configure this externally / globally with a CMake `option` or is a local variable sufficient?

Reply:
Not needed at all - see my comment above.